packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -493,6 +493,41 @@ class ShaderMask extends SingleChildRenderObjectWidget {
 /// [ImageFiltered] instead. For that scenario, [ImageFiltered] is both
 /// easier to use and less expensive than [BackdropFilter].
 ///
+/// {@tool snippet}
+///
+/// This example shows how the common case of applying a [BackdropFilter] blur
+/// to a single child can be replaced with an [ImageFiltered] widget. This code
```

Comment:
single ancestor/sibling? It's a child of the Stack that one would use, but it is an ancestor or older sibling of the BDF itself.

Reply:
Done